pkg/packages/manager.go

```diff
@@ -60,7 +61,9 @@ func processInstalling(mc *ManagerContext) bool {
 		mc.Log.Error(err, "Install failed")
 		return true
 	}
-	values[sourceRegistry] = mc.Source.Registry
+	if mc.Registry != "" && values[sourceRegistry] == "" {
```

Comment:
Is there a scenario where both of these are set? I would image if Registry is set the user would want to use that regardless of sourceRegistry being there?

Reply:
So this `values[sourceRegistry]` comes from Package and that should override everything

Reply:
So does that mean sourceRegistry from PackageBundleController spec does nothing?

Reply:
What should happen is
1. use `values[sourceRegistry]` which is from Package
2. use privateRegistry from PackageBundleController
3. use registry from bundle
4. use bundle registry from PackageBundleController

Reply:
Is there a good way to get the comment above into the code in some (preferably) obvious and readable way?

Reply:
Good idea, added a method to ManagerContext

Reply:
added some tests too